package.json

```diff
@@ -230,7 +230,7 @@
     "eslint": "7.32.0",
     "eslint-import-resolver-webpack": "^0.8.3",
     "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-depend": "^0.9.0",
+    "eslint-plugin-depend": "^0.10.0",
```

Comment:
is it really needed in scope of your PR?

Reply:
don't know how that got in there. will revert now